modules/rosetta/src/test/scala/org/tessellation/rosetta/server/JsonObjects.scala

```diff
@@ -0,0 +1,26 @@
+package org.tessellation.rosetta.server
+
+import io.circe.generic.auto._
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+
+object JsonObjects {
```

Comment:
I only created the case class for the payload because that is the only object that I need in its entirety that is common throughout all the tests.